packages/fxa-payments-server/src/routes/Product/SubscriptionCreate/index.tsx

```diff
@@ -263,6 +263,7 @@ export const SubscriptionCreate = ({
                           setSubscriptionError={setSubscriptionError}
                           ButtonBase={paypalButtonBase}
                           setTransactionInProgress={setTransactionInProgress}
+                          promoCode={coupon?.couponCode}
```

Comment:
A little confusing that a promotion code is also called a coupon code.

Reply:
updated to promotionCode